cargo-shuttle/src/provisioner_server.rs

```diff
@@ -289,6 +346,17 @@ impl Provisioner for LocalProvisioner {
         panic!("local runner should not try to delete databases");
     }
 
+    async fn provision_qdrant(
+        &self,
+        request: Request<QdrantRequest>,
+    ) -> Result<Response<QdrantResponse>, Status> {
+        let QdrantRequest { project_name } = request.into_inner();
+
+        let res = self.get_qdrant_connection_info(&project_name).await?;
```

Comment:
Could we refactor these functions so they could be used for resources that will be started similarly to qdrant in the future?

Reply:
Same as other comment